Source/ACE.Server/Managers/RecipeManager.cs

```diff
@@ -204,20 +204,19 @@ public static bool HasDifficulty(Recipe recipe)
         public static double? GetTinkerChance(Player player, WorldObject tool, WorldObject target, Recipe recipe)
         {
             // calculate % success chance
+            if (Common.ConfigManager.Config.Server.WorldRuleset != Common.Ruleset.CustomDM)
```

Comment:
Moving this to higher in method will cause these values to always be unknown (and possibly crash) if server is not set to Common.Ruleset.CustomDM